web/models.py

```
@@ -2081,6 +2085,11 @@ class Quiz(models.Model):
     """Model for storing custom quizzes created by users."""
 
     STATUS_CHOICES = [("draft", "Draft"), ("published", "Published"), ("private", "Private")]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)